test/swift_class_test.exs

```diff
@@ -101,7 +121,7 @@ defmodule SwiftClassTest do
     end
 
     test "parses key/value pairs" do
-      input = ~s|foo(bar: "baz", qux: "quux")|
+      input = ~s|foo(bar: "baz", qux: :quux")|
```

Comment:
Typo?

Reply:
in this case no, this was by design. We want to support atoms as values which we currently don't. This test covers this case.

Reply:
I should have been more clear, there's a typo on the quux. The atom is followed by a double quote